src/lib/stopTimedLoop.js

```diff
@@ -1,18 +1,24 @@
 /**
  * Stops any running timedLoops, or a specific one if passed in return value of `timedLoop`
- * @param {Number} loop The value returned by the `timedLoop()` function that you want to stop. If not included, all running timed loops will stop.
- * @todo Implement ability to stop individual loops, or all loops
+ * @param {Number/Object} loop The value returned by the `timedLoop()` function that you want to stop. If not included, all running timed loops will stop.
  */
 
 // https://stackoverflow.com/questions/8635502/how-do-i-clear-all-intervals
 // https://stackoverflow.com/questions/8860188/javascript-clear-all-timeouts
 
-const stopTimedLoop = (loop) => {
-    if(loop) {
-        if(isNaN(loop)) throw new Error(`Loop ID must be a valid integer`)
+const stopTimedLoop = (loop = false) => {
```

Comment:
`loop = false` is sort of redundant given that the conditional check is simply checking if it *is* false. 

Simply put, `if (!loop)` already accomplishes checking that its false.